plugins/dynamix/include/DashUpdate.php

```diff
@@ -185,8 +185,8 @@ function active_disks($disk) {
       }
     }
   break;
-  case 'port': exec("ifconfig -a -s|awk '/^(bond|eth|lo\s)/{print $3\"#\"$7}'",$ports); break;
-  case 'link': exec("ifconfig -a -s|awk '/^(bond|eth|lo\s)/{print \"Errors: \"$4\"<br>Drops: \"$5\"<br>Overruns: \"$6\"#Errors: \"$8\"<br>Drops: \"$9\"<br>Overruns: \"$10}'",$ports); break;
+  case 'port': exec("ifconfig -a -s|awk '/^(bond|eth|lo)[0-9]*\s/{print $3\"#\"$7}'",$ports); break;
```

Comment:
It won't break 'lo', since `[0-9]*` means with or without digit

The `\s` is now used to eliminate sub-interfaces, eg, `eth0.1`